oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -775,8 +783,9 @@ public ImpersonatedCredentials build() {
         return new ImpersonatedCredentials(this);
       } catch (IOException e) {
         // throwing exception would be breaking change. catching instead.
```

Comment:
Nit, I know this wasn't changed in the PR, but just cleaning up the comments.
```suggestion
        // Throwing an exception would be breaking change, so the exception is caught instead.
```